2023-03-27 18:17:23,258 INFO    MainThread:81 [wandb_setup.py:_flush():76] Configure stats pid to 81
2023-03-27 18:17:23,258 INFO    MainThread:81 [wandb_setup.py:_flush():76] Loading settings from /home/studio-lab-user/.config/wandb/settings
2023-03-27 18:17:23,258 INFO    MainThread:81 [wandb_setup.py:_flush():76] Loading settings from /home/studio-lab-user/Generative_Models_for_CERN_Fast_Simulations/notebooks/gan_proton_15_2312/wandb/settings
2023-03-27 18:17:23,258 INFO    MainThread:81 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-27 18:17:23,258 INFO    MainThread:81 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-03-27 18:17:23,259 INFO    MainThread:81 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-27 18:17:23,259 INFO    MainThread:81 [wandb_init.py:_log_setup():506] Logging user logs to /home/studio-lab-user/Generative_Models_for_CERN_Fast_Simulations/notebooks/gan_proton_15_2312/wandb/run-20230327_181723-zfa9l6vz/logs/debug.log
2023-03-27 18:17:23,259 INFO    MainThread:81 [wandb_init.py:_log_setup():507] Logging internal logs to /home/studio-lab-user/Generative_Models_for_CERN_Fast_Simulations/notebooks/gan_proton_15_2312/wandb/run-20230327_181723-zfa9l6vz/logs/debug-internal.log
2023-03-27 18:17:23,259 INFO    MainThread:81 [wandb_init.py:_jupyter_setup():452] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f8739cf2a30>
2023-03-27 18:17:23,259 INFO    MainThread:81 [wandb_init.py:init():546] calling init triggers
2023-03-27 18:17:23,259 INFO    MainThread:81 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {'Model': 'gan', 'dataset': 'proton_data', 'epochs': 200, 'Date': '27_03_2023_18_14', 'Proton_min': 15.0, 'Proton_max': 2133.0, 'Experiment_dir_name': 'experiments/gan_15_2133_27_03_2023_18_14'}
2023-03-27 18:17:23,259 INFO    MainThread:81 [wandb_init.py:init():568] re-initializing run, found existing run on stack: alnk7wd7
2023-03-27 18:17:23,260 INFO    MainThread:81 [wandb_run.py:_finish():1854] finishing run generative-models-cern/Generative Models for CERN Fast Simulations/alnk7wd7
2023-03-27 18:17:23,261 INFO    MainThread:81 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-27 18:17:23,261 INFO    MainThread:81 [wandb_run.py:_restore():2066] restore
2023-03-27 18:17:23,261 INFO    MainThread:81 [wandb_run.py:_restore():2072] restore done
2023-03-27 18:17:26,602 INFO    MainThread:81 [wandb_run.py:_footer_history_summary_info():3422] rendering history
2023-03-27 18:17:26,602 INFO    MainThread:81 [wandb_run.py:_footer_history_summary_info():3454] rendering summary
2023-03-27 18:17:26,604 INFO    MainThread:81 [wandb_run.py:_footer_sync_info():3380] logging synced files
2023-03-27 18:17:26,609 INFO    MainThread:81 [wandb_init.py:init():602] starting backend
2023-03-27 18:17:26,609 INFO    MainThread:81 [wandb_init.py:init():606] setting up manager
2023-03-27 18:17:26,611 INFO    MainThread:81 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-27 18:17:26,612 INFO    MainThread:81 [wandb_init.py:init():613] backend started and connected
2023-03-27 18:17:26,626 INFO    MainThread:81 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2023-03-27 18:17:26,626 INFO    MainThread:81 [wandb_run.py:_label_probe_notebook():1214] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-27 18:17:26,627 INFO    MainThread:81 [wandb_init.py:init():701] updated telemetry
2023-03-27 18:17:26,704 INFO    MainThread:81 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-03-27 18:17:27,909 INFO    MainThread:81 [wandb_run.py:_on_init():2133] communicating current version
2023-03-27 18:17:27,993 INFO    MainThread:81 [wandb_run.py:_on_init():2142] got version response 
2023-03-27 18:17:27,994 INFO    MainThread:81 [wandb_init.py:init():789] starting run threads in backend
2023-03-27 18:17:30,291 INFO    MainThread:81 [wandb_run.py:_console_start():2114] atexit reg
2023-03-27 18:17:30,291 INFO    MainThread:81 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-27 18:17:30,291 INFO    MainThread:81 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-27 18:17:30,291 INFO    MainThread:81 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-27 18:17:30,292 INFO    MainThread:81 [wandb_init.py:init():831] run started, returning control to user process
2023-03-27 18:17:30,295 INFO    MainThread:81 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-03-27 18:17:30,296 INFO    MainThread:81 [wandb_init.py:_pause_backend():417] pausing backend
2023-03-27 18:17:43,423 INFO    MainThread:81 [wandb_init.py:_resume_backend():422] resuming backend
2023-03-27 18:17:43,426 INFO    MainThread:81 [wandb_run.py:_finish():1854] finishing run generative-models-cern/Generative Models for CERN Fast Simulations/zfa9l6vz
2023-03-27 18:17:43,426 INFO    MainThread:81 [jupyter.py:save_history():445] not saving jupyter history
2023-03-27 18:17:43,426 INFO    MainThread:81 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-03-27 18:17:43,426 INFO    MainThread:81 [wandb_init.py:_jupyter_teardown():434] cleaning up jupyter logic
2023-03-27 18:17:43,427 INFO    MainThread:81 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-27 18:17:43,427 INFO    MainThread:81 [wandb_run.py:_restore():2066] restore
2023-03-27 18:17:43,428 INFO    MainThread:81 [wandb_run.py:_restore():2072] restore done
2023-03-27 18:17:46,182 INFO    MainThread:81 [wandb_run.py:_footer_history_summary_info():3422] rendering history
2023-03-27 18:17:46,182 INFO    MainThread:81 [wandb_run.py:_footer_history_summary_info():3454] rendering summary
2023-03-27 18:17:46,184 INFO    MainThread:81 [wandb_run.py:_footer_sync_info():3380] logging synced files
